app/controllers/availabilities_controller.rb

```diff
@@ -0,0 +1,17 @@
+class AvailabilitiesController < ApplicationController
+  def index
+  end
+
+  def new
+    @volunteer = Volunteer.find(params[:volunteer_id])
+  end
+
+  def create
+  end
+
+  def destroy
+  end
+
+  def update
+  end
+end
```

Comment:
It looks like you included a lot of the availabilities code in this PR for the data entries. We would need these to be in two different branches and two different pull requests.